imports/api/responses.js

```diff
@@ -321,6 +322,20 @@ Meteor.methods({
       questionId: responseObject.questionId,
       studentUserId: responseObject.studentUserId
     }, { $set: { answer: responseObject.answer, answerWysiwyg: responseObject.answerWysiwyg } })
-  }
+  },
 
+  /**
+   * Update a response item's feedback
+   * @param {Response} responseObject
+   */
+  'responses.updateFeedback' (responseObject) {
+
+    if (!responseObject) throw Error('Response not found')
+
+    let response = Responses.findOne({ _id: responseObject._id })
+    response.feedback = responseObject.feedback
```

Comment:
move this stuff to Grades. Also, need to check carefully who can edit these records.